aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -1042,5 +1025,24 @@ Tensor triton_scaled_dot_attention(const Tensor& q, const Tensor& k, const Tenso
 
 REGISTER_CUDA_DISPATCH(_fused_sdp_choice_stub, &_fused_sdp_choice_cuda);
 
+// !!This function is deprecated. See FunctionsManual.cpp for the implementation!!
```

Comment:
This could be TORCH_CHECK(false, "..."); ?

Reply:
I am afraid that if the native op is still baked into some models and this those models and if this version of libtorch is  loaded than the TORCH_CHECK would get called